package.json

```diff
@@ -59,16 +60,12 @@
     "@web-stories-wp/tracking": "*",
     "@web-stories-wp/units": "*",
     "@web-stories-wp/widget": "*",
+    "@web-stories-wp/wp-dashboard": "*",
     "@web-stories-wp/wp-story-editor": "*",
     "@wordpress/api-fetch": "^5.2.1",
-    "flagged": "^2.0.1",
-    "history": "^5.0.0",
-    "prop-types": "^15.7.2",
-    "query-string": "^7.0.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
-    "styled-components": "^5.3.0",
-    "uuid": "^8.3.2"
+    "styled-components": "^5.3.0"
```

Comment:
Where do we need this on the top level?

Reply:
Being used here, in `.storybook` 

https://github.com/google/web-stories-wp/blob/e37829642cd95eec9fe8819b5a523a9c477da256/.storybook/preview.js#L20-L21

Reply:
But probably we should move it to `devDependencies` instead.